packages/celotool/src/lib/oracle.ts

```diff
@@ -414,10 +421,13 @@ export function getOracleContextDynamicEnvVarValues<T>(
  * Reads the context and swithces to the appropriate Azure or AWS Cluster
  * Switches to the AKS cluster associated with the given context
```

Comment:
nit: update this comment -- I wouldn't be surprised if there were similar comments mentioning only Azure/AKS throughout here